src/qt/res/README.md

```diff
@@ -0,0 +1,60 @@
+# Icon policy
+This document provides guidelines for contributing icons to Bitcoin Core.
+
+
+## Preparing Icons
+Both an icon source file, in Scalable Vector Graphics (SVG) format, and an optimized production file, in Portable Network Graphics (PNG) format, are required for 
+each icon.
+
+#### SVG Source File
+SVGs are used as source files because they can scale while retaining image quality, but are not used in production due to limited application support.
+In the event that different-sized production (PNG) icons are required, they can be generated from the associated SVG source file in a vector-based tool such as:
+Inkspace, Adobe Illustrator, Figma, Sketch, or Adobe XD.

```

Comment:
Let's remove non-free software from this list. It would be annoying if we had SVGs that only render correctly in them.

Reply:
From experience SVGs exported from all of this tools are the same and interoperable. Happy to remove the adobe products though, most designers use Figma (which is free) these days anyways.

Reply:
I don't think we HAVE to remove the non-free tools, but maybe mention that Figma is the preferred tool?

Reply:
Figma is non-free and definitely not preferred...

Actually, the preferred tool (rsvg-convert) isn't even listed right now!

Reply:
I would suggest dropping "such as: Inkspace, Adobe Illustrator, Figma, Sketch, or Adobe XD" to avoid update maintenance and review effort over time on this.

Reply:
Figma is free, the paid version is only for a few extra features that do not pertain to SVG rendering. It is the preferred tool among designers (who I was pointing to in this doc as they are usually the ones who hand on the file to developers in a production sense). 

@jonatack I agree, tools come and go - it's easy enough to google and use any vector tool you wish, will make this change.

For some reference here is some info around UX/UI tools usage - https://uxtools.co/survey-2020/.

Reply:
"Free" deals with source code and licensing terms, NOT money (you're thinking "free of charge").